colors/onedark.vim

```diff
@@ -297,6 +297,17 @@ call s:h("cssSelectorOp", { "fg": s:purple })
 call s:h("cssSelectorOp2", { "fg": s:purple })
 call s:h("cssTagName", { "fg": s:red })
 
+" Dart
+call s:h("dartTypeDef", { "fg": s:red })
+call s:h("dartClassDecl", { "fg": s:red })
```

Comment:
`dartClassDecl` points to `dartStorageClass` which points to `StorageClass`, which already has highlighting in onedark.vim, so the first two are redundant